pakyow-presenter/lib/pakyow/presenter/presentable.rb

```diff
@@ -8,13 +8,27 @@ def initialize(*args)
         super
       end
 
-      def presentable(*args)
-        presentables.concat(args).uniq!
+      def presentable(name, value = nil)
+        args = {
+          name: name,
+          value: value
+        }
+
+        args[:block] = Proc.new if block_given?
```

Comment:
Correct, the only way to avoid a Proc completely is to yield here as opposed to waiting to evaluate within `render`. I think it makes the most sense to call both methods and blocks from the same place, but doing it here might be fine. I'll go ahead and start exploring that but ping me when you're around if you'd like to discuss.